torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -309,6 +315,23 @@ def aot_dispatch_autograd(
             if not hasattr(compiled_fw_func, "_boxed_call"):
                 compiled_fw_func = make_boxed_func(compiled_fw_func)
 
+            if tracing_context and tracing_context.fakify_first_call:
```

Comment:
A giant `Note [Calling AOTAutograd runtime code once with FakeTensors]` or something describing the situation and why we need it here would be really nice.

Reply:
will add when we are aligned

Reply:
+1 would be really nice to have this note

Reply:
Added